Code/Editor/Controls/ConsoleSCB.cpp

```diff
@@ -384,6 +396,30 @@ CConsoleSCB::~CConsoleSCB()
     CLogFile::AttachEditBox(nullptr);
 }
 
+void CConsoleSCB::SetupOptionsMenu()
+{
+    m_optionsMenu = new QMenu(QStringLiteral("Console Options Menu"), this);
+    connect(m_optionsMenu, &QMenu::aboutToShow, this, &CConsoleSCB::UpdateOptionsMenu);
+    ui->optionsButton->setMenu(m_optionsMenu);
+    ui->optionsButton->setAutoRaise(true);
+    ui->optionsButton->setPopupMode(QToolButton::InstantPopup);
+
+    m_clearOnPlayAction = new QAction(tr("Clear On Play"), this);
+    m_clearOnPlayAction->setCheckable(true);
+    connect(m_clearOnPlayAction, &QAction::triggered, this, [this] { ToggleClearOnPlay(); });
```

Comment:
Nit: Can connect this straight to `ToggleClearOnPlay` if you make it a slot to avoid the lambda.

Reply:
Fixed.